main.js

```diff
@@ -325,6 +325,14 @@ function showDebugLog() {
   }
 }
 
+function showSettings() {
+  if (!mainWindow) {
+    return;
+  }
+
+  mainWindow.webContents.send('show-settings');
+}
+
 function openReleaseNotes() {
   shell.openExternal(`https://github.com/signalapp/Signal-Desktop/releases/tag/v${app.getVersion()}`);
 }
```

Comment:
Since you are already working in this file and the community forum has moved to https://community.signalusers.org/ you could also change the link (https://whispersystems.discoursehosting.net/) in line 349. Unfortunately I cannot comment directly there.

Reply:
Thanks, @rainerzufall. I created a separate PR for this change: https://github.com/signalapp/Signal-Desktop/pull/2083